Makefile.dune

```diff
@@ -1,7 +1,7 @@
 # -*- mode: makefile -*-
 # Dune Makefile for Coq
 
-.PHONY: help voboot states world watch check quickbyte quickopt test-suite release apidoc ocheck ireport clean
+.PHONY: help voboot states world watch check quickbyte quickopt quickide test-suite release apidoc ocheck ireport clean
```

Comment:
I think you should start splitting this list like in `Makefile.dune`.

Reply:
I will do the split in this PR indeed, one sec.

Reply:
Well, on the other hand I like the global PHONY as it helps me keeps the help list in sync. I dunno, let me think about it. Sparse phony targets is a problem the current makefile system suffers a lot.